### DISCUSSION TITLE
support for migrating VS Code Settings

### DISCUSSION THREAD

**Discussion by sehz** [zed-industries/zed#23795] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Hi, I am currently using VSCode.  Is there quick way to migrate VSCode settings like theme?

**Comment by mikayla-maki** [comment]
Not yet, it will be added soon!

**Comment by dalisoft** [comment]
Yes, +1 to this issue.

I have my [VSCode config](https://github.com/dalisoft/vscode-config) which i'm tuned for years (300+ config) and i would not like to set again those in Zed.

**Comment by notpeter** [comment]
This is not a VSCode settings import but there is a vscode theme converter which can aid in the porting of vscode themes to zed.

```
git clone https://github.com/zed-industries/zed
cd zed/crates/theme_importer
cargo run -- convert --warn-on-missing <theme_path> -o <output_path>
```

**Comment by wiizzl** [comment]
Hello, is configuration migration possible now? Or is it still not available?

**Comment by injust** [comment]
The `zed::ImportVsCodeSettings` command was added in [v0.185.9](https://zed.dev/releases/stable/0.185.9) (https://github.com/zed-industries/zed/pull/29018).

**Reply by wiizzl** [reply]
Amazing thanks

**Reply by paddyroddy** [reply]
Any idea how to accept the risk and just embrace the settings?
![image](https://github.com/user-attachments/assets/4ad568bb-6531-405b-95d9-9ce69de11d3a)
